nix/modules/telegraf.nix

```diff
@@ -0,0 +1,71 @@
+{ config, pkgs, lib, ... }:
+let
+  monitoring =
+    if config.kuutamo.telegraf.hasMonitoring then {
+      http = {
+        url = "$MONITORING_URL";
+        data_format = "prometheusremotewrite";
+        username = "$MONITORING_USERNAME";
+        password = "$MONITORING_PASSWORD";
+      };
+    } else { };
+in
+{
+  options = {
+    kuutamo.telegraf.configHash = lib.mkOption {
```

Comment:
This is not easy to use for someone that is using the flakes directly and not using kld-mgr. Can you make it so that this can be a path the environment file with the secrets? I want to put URL, USERNAME, PASSWORD into /var/lib/secrets/telegraf and set the path to that file in this module.

Reply:
This is already discussed when implementing this on the near side, also the best practice for the system to avoid saving secrets.